src/components/Pages/OfficialDocument.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import moment from 'moment-timezone';
+
+const OfficialDocument = ({ document: { id, date, title, authoringOffice, summary, name, link }, bottomBorder, intl }) => {
+  const containerClasses = ["coa-OfficialDocumentPage__container"]
+  if (!!bottomBorder) {
```

Comment:
is this something that https://stackoverflow.com/questions/5449872/css-notlast-childafter-selector could work for?

Reply:
yep